test/models/shelter_test.rb

```diff
@@ -6,4 +6,19 @@ class ShelterTest < ActiveSupport::TestCase
     assert_equal Shelter.count, 2
   end
 
+  test "shelter name is NOT nil" do
+    shelters = Shelter.all
+    shelters.each do |shelter|
+      assert_not_equal shelter.shelter, nil
+    end
+  end
+
+  test "shelter's longitude and latitude is a float type" do
+    shelters = Shelter.all
+    shelters.each do |shelter|
+      assert_equal shelter.column_for_attribute('longitude').type, :float
```

Comment:
This is not bad, but I think you don't want to test Rails itself; testing what the models can do for you is better I think.